pages/[wallet]/[network]/[...customContract].tsx

```diff
@@ -176,6 +176,7 @@ const ContractSubnav: React.FC<ContractSubnavProps> = ({ routes }) => {
           }
         }, 10);
       }}
+      overflowX={{ base: "scroll", md: "inherit" }}
```

Comment:
can we make base "auto"? (should create scroll area if it needs it but not if not)